src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -325,7 +326,7 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
     }
 
     static validateMongoConnectionString(value: string): string | undefined {
-        if (value && value.match(/^mongodb(\+srv)?:\/\//)) {
+        if (value && value.match(/^mongodb(\+srv)?:\/\//) && !value.match(" ")) {
```

Comment:
nit: Generally you should use contains() or indexOf>=0 for a simple comparison like this since it's faster.  Won't make a lot of difference in this case, though.